.github/workflows/pgcmp.yml

```diff
@@ -0,0 +1,176 @@
+name: "Database Migrations Check PR"
+
+on:
+  pull_request:
+    paths:
+      - 'deployment/hasura/migrations/**'
+      - 'merlin-server/sql/**'
+      - 'sequencing-server/sql/**'
+      - 'scheduler-server/sql/**'
+      - 'deployment/postgres-init-db/sql/**'
+      - '.github/**' # REMOVE BEFORE MERGING, BUT THIS SHOULD FORCE THE WORKFLOW TO RUN ON THIS PR
```

Comment:
Just tagging this so it's not forgotten :)

Reply:
It's been removed!